models/base_sources/arbitrum_base_sources.yml

```diff
@@ -44,12 +44,11 @@ sources:
             description: "Whether the transaction was completed successfully"
           - &from
             name: from
-            description: "Wallet address that initiated the transaction"
+            description: 'Wallet address that initiated the transaction'
           - &to
             name: to
-            description: "Wallet address that received the transaction"
-          - &block_hash
```

Comment:
seems like dropping this line broke dbt compile.

Reply:
This is now fixed.